docs/sources/installation/helm/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Install Grafana Loki with Helm
+menuTitle: Deploy on Kubernetes
+description: Use this template when you write a concept topic.
+aliases:
+  - /docs/writers-toolkit/latest/templates/concept-template
```

Comment:
@knylander-grafana, @chri2547, and @brendamuir, please see this bug that will come up multiple times. @jdbaldry, we will need to force a build break for this alias.